### PR TITLE
CMakeLists & Setup Update

### PR DESCRIPTION
**Pull request by kiritigowda**
* Issue #354 
* Issue #353 

## CMakeList Options
* `-D NEURAL_NET=OFF` - turn off all NN modules
* `-D RALI=OFF` - turn off rali modules

## MIVisionX Setup
````
python MIVisionX-setup.py --directory [setup directory - optional (default:~/)]
                          --installer [Package management tool - optional (default:apt-get) [options: Ubuntu:apt-get;CentOS:yum]]
                          --opencv    [OpenCV Version - optional (default:3.4.0)]
                          --miopen    [MIOpen Version - optional (default:2.5.0)]
                          --miopengemm[MIOpenGEMM Version - optional (default:1.1.5)]
                          --protobuf  [ProtoBuf Version - optional (default:3.12.0)]
                          --rpp       [RPP Version - optional (default:0.4)]
                          --ffmpeg    [FFMPEG Installation - optional (default:no) [options:yes/no]]
                          --rali      [MIVisionX RALI Dependency Install - optional (default:yes) [options:yes/no]]
                          --neural_net[MIVisionX Neural Net Dependency Install - optional (default:yes) [options:yes/no]]
                          --reinstall [Remove previous setup and reinstall (default:no)[options:yes/no]]
````

### CODE REVIEW

File: MIVisionX-setup.py [hunk 1]
```diff
@@ -96,29 +103,28 @@
...
-	os.system('sudo -v')
-	os.system('(cd '+deps_dir+'/build/MIOpenGEMM; sudo '+linuxFlag+' make install -j8)')
-	os.system('sudo -v')
-	os.system('(cd '+deps_dir+'/build/MIOpen; sudo '+linuxFlag+' make install -j8)')
-	os.system('sudo -v')
-	os.system('(cd '+deps_dir+'/protobuf-'+ProtoBufVersion+'; sudo '+linuxFlag+' make install -j8)')
 	os.system('sudo -v')
 	os.system('(cd '+deps_dir+'/build/OpenCV; sudo '+linuxFlag+' make install -j8)')
-	if rppInstall == 'yes':
+	if neuralNetInstall == 'yes':
```
Comment (rrawther) [inline MIVisionX-setup.py:115]: half library is required for RALI and RPP as well

File: MIVisionX-setup.py [hunk 2]
```diff
@@ -96,29 +103,28 @@
...
+		os.system('(cd '+deps_dir+'; sudo cp half-files/include/half.hpp /usr/local/include/ )')
+		os.system('sudo -v')
+		os.system('(cd '+deps_dir+'/build/rocm-cmake; sudo '+linuxFlag+' make install -j8)')
+		os.system('sudo -v')
+		os.system('(cd '+deps_dir+'/build/MIOpenGEMM; sudo '+linuxFlag+' make install -j8)')
+		os.system('sudo -v')
+		os.system('(cd '+deps_dir+'/build/MIOpen; sudo '+linuxFlag+' make install -j8)')
+	if raliInstall == 'yes' or neuralNetInstall == 'yes':
+		os.system('sudo -v')
+		os.system('(cd '+deps_dir+'/protobuf-'+ProtoBufVersion+'; sudo '+linuxFlag+' make install -j8)')
```
Comment (rrawther) [inline MIVisionX-setup.py:126]: Protobuf is required for RALI as well

File: README.md [hunk 3]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
-  * [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
-  * [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
-  * [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
-  * [RPP Extension](amd_openvx_extensions/amd_rpp)
-  * [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
+  + [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
+  + [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
+  + [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
+  + [RPP Extension](amd_openvx_extensions/amd_rpp)
+  + [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
```
Comment (kiritigowda) [inline README.md:14]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 11
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570&bid=19777723)

File: README.md [hunk 4]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
 
 * [AMD OpenVX](#amd-openvx)
 * [AMD OpenVX Extensions](#amd-openvx-extensions)
-  * [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
-  * [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
-  * [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
-  * [RPP Extension](amd_openvx_extensions/amd_rpp)
-  * [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
+  + [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
+  + [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
```
Comment (kiritigowda) [inline README.md:11]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 5]
```diff
@@ -79,62 +82,75 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
+
 * Windows 10
 * Windows SDK
 * Visual Studio 2017 and above
 * Install the latest AMD [drivers](https://www.amd.com/en/support)
 * Install [OpenCL SDK](https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/releases/tag/1.0)
 * Install [OpenCV 3.4](https://github.com/opencv/opencv/releases/tag/3.4.0)
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
-  * Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
```
Comment (kiritigowda) [inline README.md:107]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 6]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
 
 * [AMD OpenVX](#amd-openvx)
 * [AMD OpenVX Extensions](#amd-openvx-extensions)
-  * [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
-  * [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
-  * [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
-  * [RPP Extension](amd_openvx_extensions/amd_rpp)
-  * [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
+  + [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
```
Comment (kiritigowda) [inline README.md:10]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 7]
```diff
@@ -79,62 +82,75 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
 * Windows 10
 * Windows SDK
 * Visual Studio 2017 and above
 * Install the latest AMD [drivers](https://www.amd.com/en/support)
 * Install [OpenCL SDK](https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/releases/tag/1.0)
 * Install [OpenCV 3.4](https://github.com/opencv/opencv/releases/tag/3.4.0)
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
-  * Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
+  + Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
```
Comment (kiritigowda) [inline README.md:108]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 8]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
 * [Samples](samples#samples)
 * [Toolkit](#toolkit)
 * [Utilities](#utilities)
-  * [Inference Generator](utilities/inference_generator#inference-generator)
-  * [Loom Shell](utilities/loom_shell#radeon-loomshell)
-  * [RunCL](utilities/runcl#amd-runcl)
-  * [RunVX](utilities/runvx#amd-runvx)
+  + [Inference Generator](utilities/inference_generator#inference-generator)
+  + [Loom Shell](utilities/loom_shell#radeon-loomshell)
+  + [RunCL](utilities/runcl#amd-runcl)
```
Comment (kiritigowda) [inline README.md:23]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 9]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
 * [Neural Net Model Compiler & Optimizer](#neural-net-model-compiler--optimizer)
 * [RALI](#rali)
 * [Samples](samples#samples)
 * [Toolkit](#toolkit)
 * [Utilities](#utilities)
-  * [Inference Generator](utilities/inference_generator#inference-generator)
-  * [Loom Shell](utilities/loom_shell#radeon-loomshell)
-  * [RunCL](utilities/runcl#amd-runcl)
-  * [RunVX](utilities/runvx#amd-runvx)
+  + [Inference Generator](utilities/inference_generator#inference-generator)
```
Comment (kiritigowda) [inline README.md:21]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 10]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
+  + [Loom Shell](utilities/loom_shell#radeon-loomshell)
+  + [RunCL](utilities/runcl#amd-runcl)
+  + [RunVX](utilities/runvx#amd-runvx)
```
Comment (kiritigowda) [inline README.md:24]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 11]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
 * [AMD OpenVX Extensions](#amd-openvx-extensions)
-  * [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
-  * [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
-  * [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
-  * [RPP Extension](amd_openvx_extensions/amd_rpp)
-  * [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
+  + [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
+  + [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
+  + [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
+  + [RPP Extension](amd_openvx_extensions/amd_rpp)
```
Comment (kiritigowda) [inline README.md:13]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 12]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
 * [RALI](#rali)
 * [Samples](samples#samples)
 * [Toolkit](#toolkit)
 * [Utilities](#utilities)
-  * [Inference Generator](utilities/inference_generator#inference-generator)
-  * [Loom Shell](utilities/loom_shell#radeon-loomshell)
-  * [RunCL](utilities/runcl#amd-runcl)
-  * [RunVX](utilities/runvx#amd-runvx)
+  + [Inference Generator](utilities/inference_generator#inference-generator)
+  + [Loom Shell](utilities/loom_shell#radeon-loomshell)
```
Comment (kiritigowda) [inline README.md:22]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 13]
```diff
@@ -7,21 +7,21 @@ MIVisionX toolkit is a set of comprehensive computer vision and machine intellig
...
 * [AMD OpenVX](#amd-openvx)
 * [AMD OpenVX Extensions](#amd-openvx-extensions)
-  * [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
-  * [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
-  * [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
-  * [RPP Extension](amd_openvx_extensions/amd_rpp)
-  * [WinML Extension](amd_openvx_extensions/amd_winml#amd-winml-extension)
+  + [Loom 360 Video Stitch Library](amd_openvx_extensions/amd_loomsl)
+  + [Neural Net Library](amd_openvx_extensions/amd_nn#openvx-neural-network-extension-library-vx_nn)
+  + [OpenCV Extension](amd_openvx_extensions/amd_opencv#amd-opencv-extension)
```
Comment (kiritigowda) [inline README.md:12]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 14
- Added 17
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570&bid=19777723)

File: README.md (hunk 12 above)
Comment (kiritigowda) [inline README.md:22]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 14]
```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
 
 ### Linux
+
 * Install [ROCm](https://rocm.github.io/ROCmInstall.html) 
 * ROCm CMake, MIOpenGEMM & MIOpen for Neural Net Extensions (vx_nn)
 * CMake 2.8 or newer [download](http://cmake.org/download/)
 * Qt Creator for [Cloud Inference Client](apps/cloud_inference/client_app/README.md)
 * [Protobuf](https://github.com/google/protobuf) for inference generator & model compiler
-  * install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
+  + install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
```
Comment (kiritigowda) [inline README.md:117]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 8 above)
Comment (kiritigowda) [inline README.md:23]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 15]
```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
+                          --neural_net[MIVisionX Neural Net Dependency Install - optional (default:yes) [options:yes/no]]
                           --reinstall [Remove previous setup and reinstall (default:no)[options:yes/no]]
-````
-**Note:** use `--installer yum` for **CentOS**
+```
 
-**Note:** ROCm upgrade with `sudo apt upgrade` requires the setup script rerun. 
+**Note:** 
+
+  + use `--installer yum` for **CentOS**
```
Comment (kiritigowda) [inline README.md:153]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 16]
```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
 * Install [ROCm](https://rocm.github.io/ROCmInstall.html) 
 * ROCm CMake, MIOpenGEMM & MIOpen for Neural Net Extensions (vx_nn)
 * CMake 2.8 or newer [download](http://cmake.org/download/)
 * Qt Creator for [Cloud Inference Client](apps/cloud_inference/client_app/README.md)
 * [Protobuf](https://github.com/google/protobuf) for inference generator & model compiler
-  * install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
+  + install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
 * [OpenCV 3.4](https://github.com/opencv/opencv/releases/tag/3.4.0)
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
```
Comment (kiritigowda) [inline README.md:119]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 14 above)
Comment (kiritigowda) [inline README.md:117]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 10 above)
Comment (kiritigowda) [inline README.md:24]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 17]
```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
+
 * Windows 10
 * Windows SDK
 * Visual Studio 2017 and above
 * Install the latest AMD [drivers](https://www.amd.com/en/support)
 * Install [OpenCL SDK](https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/releases/tag/1.0)
 * Install [OpenCV 3.4](https://github.com/opencv/opencv/releases/tag/3.4.0)
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
-  * Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
```
Comment (kiritigowda) [inline README.md:107]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 13 above)
Comment (kiritigowda) [inline README.md:12]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 9 above)
Comment (kiritigowda) [inline README.md:21]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 4 above)
Comment (kiritigowda) [inline README.md:11]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 6 above)
Comment (kiritigowda) [inline README.md:10]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 18]
```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
+  + Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
```
Comment (kiritigowda) [inline README.md:108]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 11 above)
Comment (kiritigowda) [inline README.md:13]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 15 above)
Comment (kiritigowda) [inline README.md:153]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md [hunk 19]
```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
...
 * [Protobuf](https://github.com/google/protobuf) for inference generator & model compiler
-  * install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
+  + install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
 * [OpenCV 3.4](https://github.com/opencv/opencv/releases/tag/3.4.0)
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
 * [FFMPEG n4.0.4](https://github.com/FFmpeg/FFmpeg/releases/tag/n4.0.4) - Optional
-  * FFMPEG is required for amd_media & mv_deploy modules
-  
+  + FFMPEG is required for amd_media & mv_deploy modules
```
Comment (kiritigowda) [inline README.md:121]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

File: README.md (hunk 3 above)
Comment (kiritigowda) [inline README.md:14]: Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570&bid=19777723)

File: README.md [hunk 20]
```diff
@@ -265,74 +309,89 @@ MIVisionX provides developers with docker images for Ubuntu 16.04, Ubuntu 18.04,
...
 runvx /opt/rocm/mivisionx/samples/gdf/canny.gdf 
-````
+```
 
 ## Release Notes
 
 ### Known issues
-* Package (.deb & .rpm) install requires **OpenCV `v3.4.0`** to execute AMD OpenCV extensions
+
+* Package (.deb & .rpm) install requires **OpenCV `v3.4.0` ** to execute AMD OpenCV extensions
```
Comment (kiritigowda) [inline README.md:388]: Codacy found an issue: [[no-inline-padding] Don’t pad `strong` with inner spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570&bid=19777723)

**Pull request merged**